### ISSUE TITLE
add custom sitemap

### ISSUE THREAD

**Issue by kingolego** [kingolego/monterey-mural#2]
Currently the site’s sitemap is auto-generated by a Jekyll module/gem; see `~/_config.yml`. The generated file can be accessed as [sitemap](https://kingolego.github.io/monterey-mural/sitemap).

This site is not large enough or subject to enough evolution to take advantage of auto-generation. In the future, we'd also like to support some additional attributes recommended for Google Search.

For more background on sitemaps, see this section on [Google Search Central](https://developers.google.com/search/docs/advanced/sitemaps/build-sitemap).

To resolve this issue:
1. Create an XML sitemap file under source control, providing the same information as the autogenerated XML file and a `<lastmod>` element for each site file. Use the extension `.xml` for both the source and deployed file, so that the new sitemap can be used in parallel with the old.
2. Deploy the updated site and manually inspect and confirm the new file is available at the new [url](https://kingolego.github.io/monterey-mural/sitemap.xml).
3. Submit a sitemap request on the [Google Search Console](https://search.google.com/search-console/sitemaps?resource_id=https%3A%2F%2Fkingolego.github.io%2Fmonterey-mural%2F). You may have to wait a week for Google to process this request. Be sure that Google indicates the new sitemap was crawled successfully. This step can be performed during integration test. If you have any problems performing this step, discuss with the author of this issue or a team lead.

After this issue is resolved, create a new issue titled "Disable the Jekyll sitemap module”, referencing this issue.

Here is the current content of the auto-generated file; it’s also on [gist](https://gist.github.com/kingolego/88a737d4fee808e90bc69c762a17c644).
```
<urlset xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
<url>
<loc>
https://kingolego.github.io/monterey-mural/faq.html
</loc>
</url>
<url>
<loc>https://kingolego.github.io/monterey-mural/</loc>
</url>
<url>
<loc>
https://kingolego.github.io/monterey-mural/scenes.html
</loc>
</url>
</urlset>
```

**Comment by kingolego** [comment]
The last time a sitemap request was submitted to Google on January 7, 2021, it took 20 days for successful processing. There was some sort of of error on Jan-8, but no details were provided. See the attached screenshot.
![image](https://user-images.githubusercontent.com/4239010/106400790-84ac4b80-63d5-11eb-8084-1bcae5d13f6c.png)

**Comment by kingolego** [comment]
Now that there are half-a-dozen pages, my thinking about this has evolved; see discussion #17. The sitemap should be based on a “golden” list of URLs that we want to expose to search bots and the outside world, but that should also be tested against what is actually built and deployed.
